docs/source/demo-apps-ios.md

```diff
@@ -1 +1,141 @@
-```{include} ../../examples/demo-apps/apple_ios/ExecuTorchDemo/README.md
+# Building an ExecuTorch iOS Demo App
+
+Welcome to the tutorial on setting up the ExecuTorch iOS Demo App!
+
+This app uses the
+[MobileNet v3](https://pytorch.org/vision/main/models/mobilenetv3.html) model to
+process live camera images leveraging three different backends:
+[XNNPACK](https://github.com/google/XNNPACK),
+[Core ML](https://developer.apple.com/documentation/coreml) and
+[Metal Performance Shaders (MPS)](https://developer.apple.com/documentation/metalperformanceshaders)
+(Xcode 15+ and iOS 17+ only).
+
+![](_static/img/demo_ios_app.jpg)
+
+## Prerequisites
+
+Before we start, make sure you have the following tools installed:
+
+### 1. Xcode 15 and Command Line Tools
+
+Install Xcode 15 from the
+[Mac App Store](https://apps.apple.com/app/xcode/id497799835) and then install
+the Command Line Tools using the terminal:
+
+```bash
+xcode-select --install
+```
+
+### 2. Python 3.10+
+
+Python 3.10 or above, along with `pip`, should be pre-installed on MacOS 13.5+.
+If needed, [download Python](https://www.python.org/downloads/macos/) and
+install it. Verify the Python and pip versions using these commands:
+
+```bash
+which python3 pip
+python3 --version
+pip --version
+```
+
+### 3. Getting Started Tutorial
+
+Follow the [Setting Up ExecuTorch](https://pytorch.org/executorch/stable/getting-started-setup)
+tutorial to configure the basic environment:
+
+```bash
+git clone -b viable/strict https://github.com/pytorch/executorch.git && cd executorch
+
+python3 -m venv .venv && source .venv/bin/activate && pip install --upgrade pip
+
+./install_executorch.sh
```

Comment:
```
./install_executorch.sh --pybind coreml mps xnnpack
```

It doesn't install MPS by default. I think it does for xnnpack and coreml now.

Reply:
@mergennachin that's weird if so, I'd expect every backend to be installed explicitly.
But anyway, no pybindings are needed for this test.

Reply:
This is due to @jathu and @larryliu0820 's  improvements in ET 0.6

We install xnnpack pybind automatically all the time. If the host is mac, then we install coreml too.